components/developer/gcc49/Makefile

```diff
@@ -99,6 +99,7 @@ CONFIGURE_OPTIONS+= --without-gnu-ld
 CONFIGURE_OPTIONS+= --with-ld=/usr/bin/ld
```

Comment:
Ok, yay! :-)
wasn't it bumped already? Or did someone else change the recipe while the
PR was posted?

--
Typos courtesy of GMail on my Samsung Android
16.06.2016 15:06 пользователь "Alexander Pyhalov" <notifications@github.com>
написал:

> In components/developer/gcc49/Makefile
> <https://github.com/OpenIndiana/oi-userland/pull/1949#discussion_r67339991>
> :
>
> > @@ -99,6 +99,7 @@ CONFIGURE_OPTIONS+= --without-gnu-ld
> >  CONFIGURE_OPTIONS+= --with-ld=/usr/bin/ld
>
> Please, bump COMPONENT_REVISION and let's merge it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/pull/1949/files/46b1dffcfedb91ee8954809f0e50adf6623724bc#r67339991>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABj5lGopQkrL9v3MQvbSc_ddbtPsm675ks5qMUphgaJpZM4IX-BX>
> .
>